instruments/technical_docs_for_high_risk_ai.yaml

```diff
@@ -0,0 +1,349 @@
+systemcard_path: .assessments[]
+schema_version: 1.1.0
+
+instrument:
+  name: "Technische documentatie voor hoog-risico AI-systemen"
+  description: "De technische documentatie van een AI-systeem met een hoog risico wordt opgesteld voordat dit systeem in de handel wordt gebracht of in gebruik wordt gesteld."
+  urn: "urn:nl:aivt:ir:td:1.0"
+  language: "nl"
+  owners:
+    - organization: "European Union"
+    - email: ""
+    - role: ""
+    - date: ""
+    - url: "https://www.europarl.europa.eu/doceo/document/TA-9-2024-0138-FNL-COR01_NL.pdf" #See article 11 and annex IV of European AI Act
+
+template: &systemcard_template                  # anchor for all fields belonging to answer
+  question: "$QUESTION"
+  answer: "$ANSWER"
+  remarks: "$REMARKS"
+  timestamp: "$TIMESTAMP"
+  authors:
+    - name: "$AUTHOR.NAME"
+      email: "$AUTHOR.EMAIL"
+      role: "$AUTHOR.ROLE"
+
+# Algemene beschrijving AI-systeem
+tasks:
+  - question: "Wat is het beoogde doel van het AI-systeem?"
+    urn: "urn:nl:aivt:ir:td:1.0:1.a.1"
+    labels:
+      - key: topic
+        value: "Algemene beschrijving AI-systeem"
+    type: "open_question"
+    suggested_roles: [ ]
+    lifecycle: [ ]
+    links: [ {"text": "DPIA vraag 4"}, {"urn": "urn:nl:aivt:ir:1.0:1.2.1"}, {"text": "system card:  goal_and_impact"}]     #TO DO: find a way to link to specific system card fields
```

Comment:
Instead of having systemcard we could maybe link to a specific systemcard version, but i see also that the system card doesn't have anurn yet [system card link](https://github.com/MinBZK/ai-validation/blob/main/docs/projects/tad/reporting-standard/0.1a3.md)